docs/document/content/user-manual/shardingsphere-proxy/distsql/syntax/rql/rule-query/single-table/count-single-table.cn.md

```diff
@@ -31,33 +31,33 @@ databaseName ::=
 
 ### 返回值说明
 
-| 列        | 说明                 |
-|-----------|---------------------|
-| rule_name | 规则名称              |
-| database  | 单表所在的数据库名称    |
-| count     | 规则个数              |
+| 列         | 说明                  |
+|------------|----------------------|
+| table_name | 单表名称              |
+| database   | 单表所在的数据库名称    |
+| count      | 单表个数              |
```

Comment:
There is no corresponding relationship between table name and count, and should not appear in the same row.

Reply:
fixed